ci/run_playbooks.yml

```
@@ -0,0 +1,31 @@
+---
+- name: "Run functional test playbooks"
+  hosts:  "{{ cifmw_target_hook_host | default('localhost')  }}"
+  gather_facts: false
```

Also needs gather facts: true 
```suggestion
  gather_facts: true
```